src/ServerRequest.php

```diff
@@ -188,7 +188,24 @@ public static function fromGlobals()
     public static function getUriFromGlobals() {
         $uri = new Uri('');
 
-        $uri = $uri->withScheme(!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' ? 'https' : 'http');
+        $https_keys = [
```

Comment:
Please follow PSR-1,2.

Reply:
Should the array be aligned? (PSRs intentionally don't have a rule about this, I personally prefer aligned arrays, but my fixer unaligned them)